app/api/items/get_breadcrumbs_from_roots.go

```diff
@@ -76,24 +78,73 @@ type breadcrumbElement struct {
 //     "$ref": "#/responses/forbiddenResponse"
 //   "500":
 //     "$ref": "#/responses/internalErrorResponse"
-func (srv *Service) getBreadcrumbsFromRoots(w http.ResponseWriter, r *http.Request) service.APIError {
+func (srv *Service) getBreadcrumbsFromRootsByItemID(w http.ResponseWriter, r *http.Request) service.APIError {
 	itemID, err := service.ResolveURLQueryPathInt64Field(r, "item_id")
 	if err != nil {
 		return service.ErrInvalidRequest(err)
 	}
 
-	user := srv.GetUser(r)
-	participantID := user.GroupID
+	return srv.getBreadcrumbsFromRoots(w, r, itemID)
+}
+
+// swagger:operation GET /items/by-text-id/{item_id}/breadcrumbs-from-roots items itemBreadcrumbsFromRootsByTextIdGet
+// ---
+// summary: List all possible breadcrumbs for a started item using `text_id`
+// description: Same as "List all possible breadcrumbs for a started item using `item_id`"
```

Comment:
Cannot you do something which would be recognized as a "link" by swagger? (don't lose 2 hours with that)

Reply:
Yes! updated